src/olympia/blocklist/tests/test_cron.py

```diff
@@ -84,49 +85,104 @@ def _block_version(
             block=block, version=version, block_type=block_type
         )
 
-    def test_skip_update_unless_force_base(self):
+    def _test_skip_update_unless_force_base(self, enable_soft_blocking=False):
         """
         skip update unless force_base is true
         """
-        upload_mlbf_to_remote_settings(force_base=False)
-
         # We skip update at this point because there is no reason to update.
+        upload_mlbf_to_remote_settings(force_base=False)
         assert not self.mocks['olympia.blocklist.cron.upload_filter.delay'].called
 
-        # But if we force the base filter, we update.
-        upload_mlbf_to_remote_settings(force_base=True)
+        filter_list = [BlockType.BLOCKED.name]
 
-        assert self.mocks['olympia.blocklist.cron.upload_filter.delay'].called
+        with override_switch('enable-soft-blocking', active=enable_soft_blocking):
+            upload_mlbf_to_remote_settings(force_base=True)
 
-        # Check that a filter was created on the second attempt
-        mlbf = MLBF.load_from_storage(self.current_time)
-        assert mlbf.storage.exists(mlbf.filter_path())
-        assert not mlbf.storage.exists(mlbf.stash_path)
+            assert (
+                mock.call(
+                    self.current_time,
+                    filter_list=filter_list,
+                    create_stash=False,
+                )
+            ) in self.mocks['olympia.blocklist.cron.upload_filter.delay'].call_args_list
 
-    def test_skip_update_unless_no_base_mlbf(self):
+            # Check that both filters were created on the second attempt
+            mlbf = MLBF.load_from_storage(self.current_time)
+            self.assertTrue(
+                mlbf.storage.exists(mlbf.filter_path(BlockType.BLOCKED)),
+            )
+            self.assertEqual(
+                mlbf.storage.exists(mlbf.filter_path(BlockType.SOFT_BLOCKED)),
+                # Until we are ready to enable soft blocking
+                # there should never be a soft block filter.
```

Comment:
So even with the switch enabled, we don't produce any sfot-block filter, right?

Reply:
Correct. The next PR, updates the tests to verify we **do** produce a soft block filter.